lib/features/home/session_model.dart

```diff
@@ -744,6 +746,23 @@ class SessionModel extends Model {
     }).then((value) => value.toString());
   }
 
+  Future<void> shareLogs() async {
+    final Directory documentDirectory =
+        await getApplicationDocumentsDirectory();
+    final String fn = '${documentDirectory.path}/logs.zip';
+    if (isDesktop()) {
+      await compute(LanternFFI.collectLogs, [fn]);
+    } else {
+      await methodChannel.invokeMethod('collectLogs', <String, dynamic>{
+        'fn': fn,
+      });
+    }
+    await Share.shareXFiles(
+      [XFile(fn)],
```

Comment:
You could maybe check the log file exists here (and throw an exception if it doesn't) before sharing // if (!await xfile.exists()) { ...